src/components/Modal/Modal.VisualTests.stories.jsx

```diff
@@ -63,3 +63,16 @@ export const FuillscreenOnMobile = () => (
     </Modal.Footer>
   </Modal>
 );
+
+export const WithMaxWidth = () => (
+  <Modal ariaLabelledBy="title" maxWidth={300} isOpen onDismiss={() => null}>
```

Comment:
Add test case for using token in addition to pixel value?

Reply:
value of `{300}` doesn't actually work. It only accepts props or css unit value as string.